src/ORM/Connect/PDOConnector.php

```diff
@@ -468,4 +496,48 @@ public function isActive()
     {
         return $this->databaseName && $this->pdoConnection;
     }
+
+    /**
+     * @inherit
+     */
+    public function transactionStart($transactionMode = false, $sessionCharacteristics = false)
+    {
+        $this->inTransaction = true;
+        if ($this->pdoConnection->beginTransaction()) {
+            if ($transactionMode) {
+                $this->query("SET TRANSACTION $transactionMode");
+            }
+            if ($sessionCharacteristics) {
+                $this->query("SET SESSION CHARACTERISTICS AS TRANSACTION $sessionCharacteristics");
+            }
+            return true;
+        }
+        return false;
+    }
+
+    /**
+     * @inherit
+     */
+    public function transactionEnd()
+    {
+        $this->inTransaction = false;
+        return $this->pdoConnection->commit();
+    }
+
+    /**
+     * @inherit
+     */
+    public function transactionRollback()
+    {
+        $this->inTransaction = false;
+        return $this->pdoConnection->rollBack();
+    }
+
+    /**
+     * @inherit
+     */
```

Comment:
You can omit these, the `@inheritDoc` tag on its own is functionally equivalent to not having a doc block at all

Reply:
:+1:

Reply:
Does removing this cause more strict linters to bark, though?

Reply:
It causes the slevomat standard to, yeah, but that'd be an issue to raise with them - this is fine in terms of the (proposed) PSR-5 spec =)

Reply:
Actually, sounds like that standard prevents you from doing this anyway: https://github.com/slevomat/coding-standard/issues/399